test/integration/tasks.test.js

```diff
@@ -396,7 +396,7 @@ describe("Tasks", function () {
     it("Should return 404 when username is invalid", function (done) {
       chai
         .request(app)
-        .get("/tasks/dummyUser?status=active")
+        .get("/tasks/dummyUser?status=in_progress")
```

Comment:
one question @prakashchoudhary07, from where are we getting this small case `active` in the constants we have `IN_PROGRESS`

Reply:
while passing it in the URL